src/deploy/functions/release.js

```diff
@@ -154,49 +154,9 @@ module.exports = function (context, options, payload) {
   var deleteReleaseNames;
   var existingScheduledFunctions;
 
-  // Collect all the functions that have a retry policy
-  var failurePolicyFunctions = functionsInfo.filter((fn) => {
-    return !!fn.failurePolicy;
-  });
-
-  let proceedPrompt = Promise.resolve(true);
-  if (failurePolicyFunctions.length) {
-    var failurePolicyFunctionLabels = failurePolicyFunctions.map((fn) => {
-      return helper.getFunctionLabel(_.get(fn, "name"));
-    });
-    var retryMessage =
-      "The following functions will be retried in case of failure: " +
-      clc.bold(failurePolicyFunctionLabels.join(", ")) +
-      ". " +
-      "Retried executions are billed as any other execution, and functions are retried repeatedly until they either successfully execute or the maximum retry period has elapsed, which can be up to 7 days. " +
-      "For safety, you might want to ensure that your functions are idempotent; see https://firebase.google.com/docs/functions/retries to learn more.";
-
-    utils.logLabeledWarning("functions", retryMessage);
-
-    if (options.nonInteractive && !options.force) {
-      throw new FirebaseError("Pass the --force option to deploy functions with a failure policy", {
-        exit: 1,
-      });
-    } else if (!options.nonInteractive) {
-      proceedPrompt = promptOnce({
-        type: "confirm",
-        name: "confirm",
-        default: false,
-        message: "Would you like to proceed with deployment?",
-      });
-    }
-  }
-
   delete payload.functions;
 
-  return proceedPrompt
-    .then((proceed) => {
-      if (!proceed) {
-        throw new FirebaseError("Deployment canceled.", { exit: 1 });
-      }
-
-      return Promise.resolve(context.existingFunctions);
-    })
+  return Promise.resolve(context.existingFunctions)
```

Comment:
This is pretty ugly right now - i left it like this because I've got  full refactor of this file coming and didn't want to clean up something that was going away.